cmd/version_test.go

```diff
@@ -34,8 +35,8 @@ func TestVersion(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !strings.Contains(output, cmd.VERSION) {
-		t.Fatal("Output does not contain version")
+	if !strings.Contains(output, cmd.VERSION) && !strings.Contains(output, cmd.CONTROLLERVERSION) {
```

Comment:
What are the values of `cmd.VERSION` and `cmd.CONTROLLERVERSION` here? Does `cmd.VERSION` default to `vlatest` and `cmd.CONTROLLERVERSION` to `latest`?